slices/libavutil58.yaml

```diff
@@ -0,0 +1,23 @@
+package: libavutil58
+
+essential:
+  - libavutil58_copyright
+
+slices:
+  libs:
+    essential:
+      - libc6_libs
+      - libdrm2_libs
+      - libva-drm2_libs
+      - libva-x11-2_libs
+      - libva2_libs
+      - libvdpau1_libs
+      - libvpl2_libs
+      - libx11-6_libs
+      - ocl-icd-libopencl1_libs
```

Comment:
![image](https://github.com/user-attachments/assets/b40c5b94-5e5a-42b2-b2ab-98f5b1e1c8a6)
Those dependencies are all covered by the `ocl-icd-libopencl1` packags, see:
![image](https://github.com/user-attachments/assets/883a3b7d-ecbd-4c1f-a04e-6a9998d0ce23)